CMakeLists.txt

```diff
@@ -30,6 +30,9 @@ if(MSVC)
     add_definitions("-WX -W2")
 else()
     add_definitions("-Wall -Wextra -Werror")
+    # workaround for
+    # error: taking address of packed member of in mavlink include
+    add_definitions("-Wno-address-of-packed-member")
```

Comment:
Why do we need to add this here, as there is no direct include of mavlink from this project?

Reply:
There is a direct include when we use MAVLink passthrough.

Reply:
The include still happens indirectly through the SDK. I prefer the SDK to handle this and a more local solution instead of globally disabling the warning. Something like this:
```
#pragma GCC diagnostic push
#if defined(__GNUC__) && __GNUC__ >= 9
#pragma GCC diagnostic ignored "-Waddress-of-packed-member"
#endif
#include "mavlink/v2.0/common/mavlink.h"
#pragma GCC diagnostic pop
```

Reply:
Thanks, I'll try that.

Reply:
It actually should already ignore warnings for this include (if it is included correctly):

https://github.com/mavlink/MAVSDK/blob/d83f57f9d2ac02b7eb48cd0c95b1df6f1d911907/src/core/mavlink_include.h#L3-L5

Reply:
Change coming up: https://github.com/mavlink/MAVSDK/pull/812